cmd2/parsing.py

```diff
@@ -297,6 +298,11 @@ def parse(self, rawinput: str) -> Statement:
             pipe_pos = tokens.index(constants.REDIRECTION_PIPE)
             # save everything after the first pipe as tokens
             pipe_to = tokens[pipe_pos+1:]
+
+            for pos, cur_token in enumerate(pipe_to):
+                unquoted_token = utils.strip_quotes(cur_token)
+                pipe_to[pos] = os.path.expanduser(unquoted_token)
```

Comment:
It seems like we do need to do something vaguely like this for pipes as well.  But since we are piping to a **process** and not **redirecting** to a file, I don't think this is exactly what we should be doing.

By doing this I think we might be eliminating the ability to pipe to an executable which isn't backed by a file on the user's PATH.  For example, can we pipe to commands built into the shell (Bash, cmd.exe, etc)?

Built-in Bash commands are listed here:
http://manpages.ubuntu.com/manpages/xenial/man7/bash-builtins.7.html